models/organizer.json

```diff
@@ -97,7 +97,7 @@
       "type": "string",
       "example": "https://www.publiq.be",
       "description": "The unique website of the organizer.",
-      "pattern": "^http[s]?:\\/\\/"
+      "format": "uri"
```

Comment:
Should we follow this more consistent? I think we have a mix now of the regex and uri.

Reply:
Yes, I will just do it now and update any tests in udb3-silex / udb3-acceptance-tests so it doesn't stay inconsistent for too long